tests/bootstrap.js

```diff
@@ -65,6 +70,24 @@ const waitForPg = async (url) => {
   }});
 };
 
+const isDBReady = (url = '') => {
+  const http = url.indexOf('https') > -1 ? require('https') : require('http');
```

Comment:
This is a fairly rough way to check for which protocol to use. Consider this instead:

```suggestion
  const urlIMPORTED = require('url'); // put this at the top of the file
  
  const parsed = new urlIMPORTED.URL(url);
  const http = parsed.protocol === 'https:' ? require('https') : require('http');
```

Obviously `urlIMPORTED` is a terrible name so you may way the change the parameter to something else so you can call the package `url`. - I'll leave this up to you!

Reply:
This is better. Done!